common/djangoapps/edxmako/shortcuts.py

```diff
@@ -99,6 +99,17 @@ def microsite_footer_context_processor(request):
     )
 
 
+def custom_theme_context_processor(request):  # pylint: disable=unused-argument
+    """
+    Sets a boolean context variable to indicate whether the site uses a custom theme.
+    """
+    return dict(
+        [
+            ("THEME_ENABLED", settings.FEATURES.get("USE_CUSTOM_THEME", False))
+        ]
+    )
+
+
```

Comment:
I wrote it that way to make it consistent with other functions in that file. Your comment encouraged me to change them all to use literal dicts.